tests/test_base_client.py

```diff
@@ -144,6 +155,18 @@ def test_failure_PUT_422(self):
             err = e.exception.error
             self.assertEqual(err.type, "validation")
 
+    def test_DELETE_204(self):
+        with delete_resource_client() as conn:
+            client = MockClient("subdomain", "apikey")
+            resource = client.delete_resource("123")
+            # conn.request.assert_called_with(
+            #     "DELETE",
+            #     "/resources/123",
+            #     None,
+            #     headers=expected_headers,
+            # )
```

Comment:
What's the plan with all this stuff that's commented out? Is there a ticket to address these?